gitclone/tracker/tracker.go

```diff
@@ -1,4 +1,4 @@
-package gitclone
+package tracker
```

Comment:
I had to move this to its own subpackage because I introduced an import cycle (by using the tracker not only in `gitclone`, but in `gitclone/bitriseapi` too)